src/sdk.ts

```diff
@@ -6,10 +6,24 @@ import { Interfaces } from './types'
 
 const pExec = util.promisify(childProcess.exec)
 
+export interface ExecOptions {
+  spawn?: boolean
+  env?: any
```

Comment:
could we be more specific here about the type of `env`? would it be something like this, maybe?

```
interface Env {
    [key: string]: string | undefined
}
```